src/Tools/dotnet-counters/Program.cs

```diff
@@ -42,7 +42,15 @@ internal class Program
                 // Handler
                 HandlerDescriptor.FromDelegate((ExportDelegate)new CounterMonitor().Collect).GetCommandHandler(),
                 // Arguments and Options
-                CounterList(), ProcessIdOption(), RefreshIntervalOption(), ExportFormatOption(), ExportFileNameOption()
+                CounterList(), ProcessIdOption(), RefreshIntervalOption(), ExportFormatOption(), ExportFileNameOption(), NameOption()
+            };
+
+        private static Option NameOption() =>
```

Comment:
It is too bad that we can't make the help text generic enough to put this in Tools\Common\Commands and share this across all the tools.

Reply:
Never mind.  It looks like the tools don't share the ProcessId commands and this is very little code.

Reply:
Yes, I wanted to share this part of the code too but this was as far as I could go without having to go through a major refactoring of all the tools :(.

Reply:
Is the help text the only thing preventing this code from being shared?  If so, I think it's worth the small price to pay for code sharing.  You could also wrap the `Option` constructor with a static method that takes a `string` that would be used in the help text.